netbox/utilities/templatetags/form_helpers.py

```diff
@@ -51,8 +51,11 @@ def widget_type(field):
 def render_fieldset(form, fieldset):
     """
     Render a group set of fields.
+
+    The signature for row tuples is (layout, title, items, has_field_errors).
```

Comment:
This is pretty firmly beyond the scope of the bug fix. If needed, let's capture this work under a separate issue, as changes to these general-purpose tools affect the entire application.

Reply:
I was mistaken that this was even needed. When I first tried to address this, the simple template modification for just InlineFields was not working without the larger change. After removing the changes to `render_fieldset()`, they still worked--which surprised me. I clearly had some other issue there that I subsequently fixed without realizing it.